lib/iris/fileformats/nimrod_load_rules.py

```diff
@@ -399,41 +399,48 @@ def set_british_national_grid_defaults(field):
 
 
 def coord_system(field):
-    """Define the coordinate system for the field."""
+    """Define the coordinate system for the field.
+    Handles Transverse Mercator, Universal Transverse Mercator and Plate Carree.
+
+    Transverse Mercator projections will default to the British National Grid if any
+    parameters are missing.
+    """
     ellipsoid = proj_biaxial_ellipsoid(field)
 
     if field.horizontal_grid_type == 0:
         # Check for missing grid meta-data and insert OSGB definitions.
         # Some Radarnet files are missing these.
         set_british_national_grid_defaults(field)
     if field.horizontal_grid_type == 0 or field.horizontal_grid_type == 4:
-        coord_sys = iris.coord_systems.TransverseMercator(
+        crs_args = (
             field.true_origin_latitude,
             field.true_origin_longitude,
             field.true_origin_easting,
             field.true_origin_northing,
             field.tm_meridian_scaling,
-            iris.coord_systems.GeogCS(**ellipsoid),
+        )
+        if any([is_missing(field, v) for v in crs_args]):
+            warnings.warn(
+                f"Coordinate Reference System is not completely defined. "
+                f"Plotting and reprojection may be impaired."
+            )
+        coord_sys = iris.coord_systems.TransverseMercator(
+            *crs_args, iris.coord_systems.GeogCS(**ellipsoid),
         )
     elif field.horizontal_grid_type == 1:
         coord_sys = iris.coord_systems.GeogCS(**ellipsoid)
     else:
-        raise TranslationError(
```

Comment:
I can't clearly see the context, but I think it is because this Translation Error is triggered by another function and doesn't need to be in the code-base twice.

Reply:
That's fine